### PR TITLE
Fix: Import resolver page is being displayed on empty import conflict [ED-8106]

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by davseve — APPROVED** [review]

File: core/app/modules/import-export/assets/js/pages/import/import-content/import-content.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ export default function ImportContent() {
 			return (
 				<ImportContentFooter
 					hasPlugins={ ! ! plugins.length }
-					hasConflicts={ ! ! ( includes.includes( 'templates' ) && uploadedData?.conflicts ) }
+					hasConflicts={ ! ! ( includes.includes( 'templates' ) && uploadedData?.conflicts && Object.keys( uploadedData.conflicts ).length ) }
```
Comment (mark-elementor) [inline core/app/modules/import-export/assets/js/pages/import/import-content/import-content.js:38]:
```suggestion
					hasConflicts={ ! ! ( includes.includes( 'templates' ) && uploadedData?.conflicts && Object.keys( uploadedData?.conflicts || {} ).length ) }
```

File: core/app/modules/import-export/assets/js/pages/import/import-process/import-process.js [hunk 2]
```diff
@@ -139,7 +139,7 @@ export default function ImportProcess() {
 				if ( kitState.data?.manifest?.plugins || importContext.data.uploadedData?.manifest.plugins ) {
 					importContext.dispatch( { type: 'SET_PLUGINS_STATE', payload: 'have' } );
 				}
-				if ( uploadedData.conflicts && ! isResolvedData ) {
+				if ( uploadedData.conflicts && Object.keys( uploadedData.conflicts ).length && ! isResolvedData ) {
```
Comment (mark-elementor) [inline core/app/modules/import-export/assets/js/pages/import/import-process/import-process.js:142]:
```suggestion
				if ( uploadedData.conflicts && Object.keys( uploadedData.conflicts || {} ).length && ! isResolvedData ) {
```

**Review by mark-elementor — CHANGES_REQUESTED** [review]

File: core/app/modules/import-export/assets/js/pages/import/import-content/import-content.js (hunk 1 above)
Comment (ManorHazaz) [reply core/app/modules/import-export/assets/js/pages/import/import-content/import-content.js:38]: Why? I already checked if the conflicts exited.

File: core/app/modules/import-export/assets/js/pages/import/import-process/import-process.js (hunk 2 above)
Comment (ManorHazaz) [reply core/app/modules/import-export/assets/js/pages/import/import-process/import-process.js:142]: Same.

**Review by mark-elementor — APPROVED** [review]

**Pull request merged**